assets/stylesheets/contributions.scss

```diff
@@ -169,6 +169,46 @@ $control-spacing: 5px;
     }
 }
 
+.contributions__banner--mother-load {
+    background-image: url('https://media.guim.co.uk/e7b4b3b6cd9a972b1cee26ed4c53d0b38fedb36d/0_268_2480_1408/2480.jpg');
```

Comment:
probably not worth it this time, but in future, you could do this as an `<img>` and use `srcset` to load lower resolutions on mobile devices